src/main/java/io/appium/java_client/Location.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client;
+
+/**
+ * Represents the physical location.
+ */
+public class Location extends org.openqa.selenium.html5.Location {
```

Comment:
would this data model remain in selenium?

Reply:
it's deprecated, so it will be removed eventually, but for now we need to keep the inheritance for backward compatibility

Reply:
makes sense. Is `org.openqa.selenium.html5.Location` marked for deprecation? I am just thinking how we tell users to switch to the new location class? Selenium does know anything about our `Location` class

Reply:
yes, `org.openqa.selenium.html5.Location` is marked as deprecated, but not released yet